Documentation/Storage-Configuration/NFS/nfs-csi-driver.md

```diff
@@ -5,8 +5,7 @@ title: CSI provisioner and driver
 !!! attention
     This feature is experimental and will not support upgrades to future versions.
 
-In version 1.9.1, Rook is able to deploy the experimental NFS Ceph CSI driver. This requires Ceph
-CSI version 3.7.0 or above. We recommend Ceph v16.2.7 or above.
+We recommend Ceph v16.2.7 or above.
```

Comment:
Let's even remove this. The default is v17.2.6, people don't really downgrade

Reply:
done, ptal thanks.